### DISCUSSION TITLE
Definitive way to setup docksal on windows 11

### DISCUSSION THREAD

**Discussion by timcorkerton** [docksal/docksal#1597] [Q&A]
Hi

I love docksal and it works great on my ubuntu machine, however I now need to develop within windows 11.

Is there a definitive (preferred) approach to installing and using docksal on windows 11.

There are so many conflicting articles on the web. 

Do we

1. Use only WSL2 with Ubuntu 20.04 for instance
2. Install Docker Desktop / Docker for windows
3.  Use VirtualBox

I'd prefer to use phpStorm but may need to accept that VSCode is the easier option

I need to use XDEBUG from my IDE

Many thanks
Tim

**Comment by lmakarov** [comment]
WSL1 with VirtualBox would be the most stable mix on Windows for the time being.

Docker Desktop keeps changing and breaking things and [WSL2 support](https://github.com/docksal/docksal/issues/1055) is unfortunately not yet there.

None of the Docksal maintainers use Windows as their daily driver, so it's always a challenge to support Windows.
The hope is that WSL2 will make it a much more streamlined experience for both users and maintainers.
